pkg/cluster/internal/create/actions/installcni/cni.go

```diff
@@ -122,23 +122,6 @@ func (a *action) Execute(ctx *actions.ActionContext) error {
 		return errors.Wrap(err, "failed to apply overlay network")
 	}
 
-	// add cni placeholder to trick the CRI so it consider the network is ready
-	// this way kubelet will not set the Node.Status to NotReady
-	cniPlaceholder := `
-{
-  "cniVersion": "0.3.1",
-  "name": "kindnet",
-  "plugins": [
-	{ "type": "portmap" }
-  ]
-}`
-
-	if err := node.Command(
-		"cp", "/dev/stdin", "/etc/cni/net.d/10-kindnet.conflist",
-	).SetStdin(strings.NewReader(cniPlaceholder)).Run(); err != nil {
```

Comment:
this is interesting @BenTheElder , adding the placeholder the kubelet is ready soon but the pods take more time, etcd takes a lot of time to be ready ... I don't know if there is some logic to give priority to hostNetwork pods or something if there are no network?